app/factory.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
```

Comment:
May be the case of renaming the `factory` => `app-factory` to be more accurate.

Reply:
As discussed, I think I'll rename the `factory.js` to `create_app.js` or similar.